### PR TITLE
Fix iOS VoiceOver so that --exposure updates

### PR DESCRIPTION
**Pull request by WestonThayer**
iOS VoiceOver with Safari (tested on iOS 15.1) has special gestures when it's virtual cursor is on a slider, you can swipe up/down to increment/decrement the slider. In this case, the input's "input" event does not fire, it only receives "change" after each gesture. This change simply listens for the "change" event too.

This also fixes macOS VoiceOver's interaction if you interact entirely with its virtual cursor (probably uncommon, you have to use SHIFT + VO + DOWN to move cursor to the slider thumb, then VO + LEFT/RIGHT to increment/decrement).

Note that iOS VoiceOver and Android Talkback both support a double-tap-then-hold, wait, then drag gesture to directly manipulate sliders. That fires "input" events. Both have issues announcing progress while dragging and when finished dragging (for Talkback, the issue is with https://bugs.chromium.org/p/chromium/issues/detail?id=1272107), but those are issues with the native range input.

### CODE REVIEW

File: src/index.js
Comment (Paul-Hebert) [inline src/index.js:187]: @WestonThayer thank you for this PR! 

I didn't realize that iOS VoiceOver didn't fire input events when using the range input this way. That's a great reminder to test using a variety of assistive devices.

I'm a little confused about line 186 and 187:

- Where is `handleChange` being defined?
- Are these necessary in addition to the event listeners added in the loop above?

Thanks!

Comment (WestonThayer) [reply src/index.js:187]: Oh no! Those aren't supposed to be there. Remnants from before I switched to using a `['input', 'change']` array. Removing now...

Comment (WestonThayer) [reply src/index.js]: Ok gone 😅, thanks for catching it.

Yes, it is quirky. I suspect it's a quirk with Safari/WebKit. I'd file a bug there, but I don't think there's a spec to cover what events an input should fire when incremented/decremented via the OS accessibility APIs... issues like this add weight to the argument that there should be though.

Comment (Paul-Hebert) [reply src/index.js]: Thanks! I'll test this out later today or tomorrow and get this merged in 🙂

Comment (WestonThayer) [reply src/index.js]: Sounds good! Note that at least on iOS 15.0.2, Safari wasn't firing *any* events when using VoiceOver with the range input, so check your version if it's not working.

Comment (Paul-Hebert) [reply src/index.js]: >Note that at least on iOS 15.0.2, Safari wasn't firing any events when using VoiceOver with the range input

Oh wow! VoiceOver wasn't firing `change` or `input`? That definitely seems buggy!

Comment (WestonThayer) [reply src/index.js]: Agreed, I was glad to find it at least partially fixed in iOS 15.1.

I opened https://bugs.webkit.org/show_bug.cgi?id=233449

**Review by Paul-Hebert — APPROVED** [review]
This works great! Thank you so much for this PR @WestonThayer, I appreciate it!

**Comment by Paul-Hebert** [comment]
@WestonThayer feel free to squash and merge this when you get a chance, or if you'd prefer I can merge it in. Thanks!

**Comment by WestonThayer** [comment]
Whoops wrong button. @Paul-Hebert I think you'll have to merge, I don't have write access to the repo.

**Comment by Paul-Hebert** [comment]
Oh of course, I forgot about that. I'll merge this in now. Thanks Weston!

**Pull request merged**
